### DISCUSSION TITLE
What is the difference between temporary and persistent limits?

### DISCUSSION THREAD

**Discussion by micw** [tbnobody/OpenDTU#683] [Q&A] [answered]
Hello,

the UI allows to set a persistent or temporary limit to the inverter. What exactly is the meaning of persistent/temporay in this context?

Especially I'd like to know if I can change a persistent limit after I set it or if it's somehow locked (e.g. to permanently limit an inverter to a certain output).

**Comment by DPO99** [comment, marked as answer]
Hi, you can find the answers [here](https://github.com/tbnobody/OpenDTU/blob/master/docs/MQTT_Topics.md).
Scroll down.

BR
Timo

**Comment by micw** [comment]
Great, thanks a lot for clarification.

This info should probably go to the GUI and not be buried in the MQTT docs ^^

**Reply by jaybee79** [reply]
Please be aware: doing persistant settings over and over again will damage the memory position, the inverter might become unuseable.
If you want to change the power settings often, use the non-persistant limits! Of cause, they are lost after a power cycle but as they are only in RAM you can't damage the inverter using them.

br
Jürgen

**Reply by DPO99** [reply]
@tbnobody ist es möglich diese Info in den mqtt topics mit aufzunehmen? Ich denke das ist gerade für nicht Profis (wie mich) eine sehr wertvolle Info. Danke.
